packages/expect/src/__tests__/matchers-toStrictEqual.property.test.ts

```diff
@@ -46,4 +56,18 @@ describe('toStrictEqual', () => {
       assertSettings,
     );
   });
+
+  it('should be equivalent to Node deepStrictEqual', () => {
+    fc.assert(
+      fc.property(
+        fc.anything(anythingSettings),
+        fc.anything(anythingSettings),
+        (a, b) => {
+          expect(safeExpectStrictEqual(a, b)).toBe(
+            safeAssertDeepStrictEqual(a, b),
+          );
+        },
```

Comment:
Maybe add `assertSettings` as I did for the others. It makes sure we use the same configuration for all the calls to `fc.assert`.

By the way, I recently introduced a way to configure in a single place the parameters that will be used by `fc.assert` - see [`fc.configureGlobal`](https://github.com/dubzzz/fast-check/blob/master/documentation/1-Guides/Tips.md#setup-global-settings) (Available since v1.18.0 - https://github.com/dubzzz/fast-check/pull/439). We might get rid off the need for `assertSettings` with this new feature.

Reply:
done